tests/plugins/test_orchestrate_build.py

```diff
@@ -364,6 +364,7 @@ def test_orchestrate_build(tmpdir, caplog, config_kwargs,
     if config_kwargs is not None:
         expected_kwargs.update(config_kwargs)
     expected_kwargs['build_image'] = 'some_image:latest'
+    expected_kwargs['reactor_config_override'] = reactor_dict
```

Comment:
`reactor_config_override` is set in both cases?

Reply:
yes because even when we aren't using reactor config map we still "are using it for clusters"

Reply:
aha! So is the check for empty really not a valid use case because reactor_config will never be empty?

Reply:
if clusters will be None it will fail in reactor_config on NonType not having items,
so I've added use case in unkown_platform to provide empty clusters {}